setup.py

```diff
@@ -18,8 +18,8 @@ def read(*parts):
 
 
 def get_version():
-    return re.findall("__version__ = '([\d\.]+)'",
-                      read('mozdownload', 'cli.py'), re.M)[0]
+	p = re.compile(r"__version__ = '(\d.+)'")
```

Comment:
What was basically requested is to get rid of `\d` by replacing it with `0-9`, and not changing all the code around the regex. Please get this reverted/updated.

Reply:
**I was thinking of converting** 
    return re.findall("__version__ = '([\d\.]+)'",
                      read('mozdownload', 'cli.py'), re.M)[0]
**to a raw string**
    return re.findall(r"__version__ = '([\d\.]+)'",
                      read('mozdownload', 'cli.py'), re.M)[0]
Will that work?
Help appreciated.

Reply:
That should work too. Make sure to be able to reproduce the deprecation warning first with Python3. Then it makes it easier for you to verify the changes.